mpf/devices/autofire.py

```diff
@@ -40,6 +42,11 @@ def __init__(self, machine: "MachineController", name: str) -> None:
         super().__init__(machine, name)
         self.delay = DelayManager(self.machine.delayRegistry)
         self._ball_search_in_progress = False
+        if self.config['enable_timeouts']:
```

Comment:
The problem here is that config is set after __init__ is done. I guess all you have to do is move this to _initialize which is called after self.config is set.

Reply:
It actually had been in _initialize before and I was still getting the same error. I actually moved it to __init__ because linter was complaining about the variable declarations being outside of __init__ in that error log you posted. I'll try moving it back into _initialize but unfortunately I don't think that's it. It's probably something stupid like I put the options in the wrong config file.